core/unit_test/TestComplex.hpp

```diff
@@ -688,6 +687,24 @@ TEST(TEST_CATEGORY, complex_structured_bindings) {
   test.testit();
 }
 
+constexpr bool basics_are_constexpr() {
```

Comment:
What does "basics" mean?

Reply:
I've renamed it `construction_is_constexpr`, OK with you ?